lib/hummingbird_survey/models/survey_page_form.rb

```diff
@@ -153,14 +153,23 @@ def build_final_page_data
       result
     end
 
+    def answer_value(value, item)
+      case item.survey_itemable.question_type
+      when SurveyQuestionType::Encrypted.new
+        EncryptionService.encrypt(value)
```

Comment:
@cgratigny : Since there isn't a dragonfly PR for this, might be helpful to have a comment here to remind us to implement that service in Dragonfly when the time comes.